filter.imp

```diff
@@ -96,21 +96,11 @@
 
 
    # The below headers were added by hand due to errors being found when the private headers were included.
-   { include: ["@.asm/bitsperlong.h.", private, "<linux/bits.h>", "public"] },
-   { include: ["@.asm/errno.h.", private, "<linux/errno.h>", "public"] },
-   { include: ["@.asm/resource.h.", private, "<linux/resource.h>", "public"] },
+   # Additionally, If an asm header has no asm-generic version like pgtable_64_types we can use a
+   # non-architecture specific header instead
```

Comment:
ptrace.h doesn't have an asm-generic version, but it's not in this table in filter.imp but rather nonarch.imp.

I still think we need to refine //why// a rule should go here vs nonarch.imp.  The rule of thumb still isn't precise enough I think.